src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -3184,5 +3177,222 @@ public async Task DynamicTableSizeUpdate_Exceeds_Settings_Throws()
                     await con.ShutdownIgnoringErrorsAsync(streamId);
                 });
         }
+
+        [Fact]
+        public async Task DynamicTable_Reuse_Concat()
+        {
+            await Http2LoopbackServer.CreateClientAndServerAsync(
+                async uri =>
+                {
+                    using HttpClient client = CreateHttpClient();
+                    using HttpResponseMessage response = await client.GetAsync(uri);
+
+                    Assert.True(response.Headers.TryGetValues("my-header", out IEnumerable<string> values));
+                    Assert.True(values.SequenceEqual(new[] { "foo", "bar" }));
+                },
+                async server =>
+                {
+                    byte[] frameData = new byte[16 * 1024];
+
+                    Http2LoopbackConnection con = await server.EstablishConnectionAsync();
+                    int streamId = await con.ReadRequestHeaderAsync();
+
+                    int pos = 0;
+                    pos += HPackEncoder.EncodeHeader(":status", "200", HPackFlags.None, frameData.AsSpan(pos));
+                    pos += HPackEncoder.EncodeHeader("my-header", "foo", HPackFlags.NewIndexed, frameData.AsSpan(pos));
+                    pos += HPackEncoder.EncodeHeader(62, "bar", HPackFlags.None, frameData.AsSpan(pos));
```

Comment:
Where is the `62` coming from?

Reply:
Oh, is this because there are 62 entries in the static table, 0-indexed?  If so, can you create a named const for this somewhere?

Reply:
sure

Reply:
Added `HPackEncoder.LargestStaticIndex` and `HPackEncoder.SmallestDynamicIndex`.